src/compiler/commandLineParser.ts

```diff
@@ -1499,6 +1496,14 @@ namespace ts {
         }
     }
 
+    export function getErrorForNoInputFiles({ includeSpecs, excludeSpecs }: ConfigFileSpecs, configFileName?: string) {

```

Comment:
:+1: `x: string | undefined` is better than `x?: string` as it ensures that we don't forget to provide the parameter.